### DISCUSSION TITLE
시험 신청하신 분들 함께 일정 공유해욥!

### DISCUSSION THREAD

**Discussion by justinyoo** [fusiondevkr/pl900#40] [0 공지사항]
24일 Virtual Training Day 과정 다들 신청하셨죠? 아직 안하신 분들은 [클릭](https://aka.ms/fdk/pl900/vtd)!

우리 댓글로 다함께 시험날짜 공유해 볼까요?

일단 저는 8월 26일 14.15 입니다!

**Comment by sesme100** [comment]
8월 24일 Virtual Training Day 과정을 수강 하시면 몇 일 내에 시험 등록을 할 수 있는 Voucher가 이메일로 전송됩니다.
확인 후 꼭 시험 등록하시고 여기에 시험 날짜 공유해주세요!! 

Voucher로 PL-900  시험 신청 방법 
https://www.youtube.com/watch?v=FOFWbSYbbVI

**Comment by justinyoo** [comment]
전 합격했습니다!

**Comment by coogie02** [comment]
저는 8월 29일 3시 45분 입니다.

**Reply by justinyoo** [reply]
고생하셨습니다! 잘 보셨겠쬬오오오?
